data/cameras.xml

```diff
@@ -17159,6 +17159,24 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
+	<Camera make="Hasselblad" model="X2D 100C">
+		<ID make="Hasselblad" model="X2D 100C">Hasselblad 100-20-Coated6</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="126" y="94" width="-118" height="0"/>
```

Comment:
Same, crop still leaves a few garbage pixels in.
How do you come up with it? Have you tried the methodology i outlined in some previous PR?

Reply:
Sorry, I indeed had no time to go through the zoom & scroll procedure, this was from raw data PPM dump...